data/src/test/java/org/oppia/data/persistence/PersistentCacheStoreTest.kt

```diff
@@ -61,10 +64,9 @@ class PersistentCacheStoreTest {
 
   @Inject lateinit var dataProviders: DataProviders
 
-  @ExperimentalCoroutinesApi
-  @Inject
-  @field:TestDispatcher
-  lateinit var testDispatcher: TestCoroutineDispatcher
+  @InternalCoroutinesApi @Inject lateinit var testCoroutineDispatchers: TestCoroutineDispatchers
+
+  @Inject @field:BackgroundDispatcher lateinit var backgroundDispatcher: CoroutineDispatcher
```

Comment:
Why are we not using the TestCoroutineDispatchers' backgroundDispatcher here?

Reply:
Technically they're the same thing, so I just arbitrarily chose to use the same code as production uses in this case.